app/templates/home-view.jade

```diff
@@ -216,7 +231,7 @@ block content
                 p.text-navy(data-i18n="new_home.for_teachers_subblurb4")
         if me.isAnonymous()
           .text-center
-            button.btn.btn-lg.btn-primary.signup-button.signup-home-btn(data-event-action="Homepage Sign Up #3 CTA", data-i18n="signup.sign_up")
+            button.btn.btn-lg.btn-primary.teacher-btn(data-event-action="Homepage Sign Up #3 CTA", data-i18n="signup.sign_up")
```

Comment:
Is this the minor change that you mention in the comments? (The one with the modal)

Reply:
Yes thats correct